Failed to build voting service

```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Conference :: Vote ................................. SUCCESS [  0.469 s]
[INFO] Conference :: Vote :: Application .................. SUCCESS [  3.486 s]
[INFO] Conference :: Vote :: Deploy ....................... SUCCESS [  0.011 s]
[INFO] Conference :: Vote :: Config ....................... FAILURE [  3.868 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 8.297 s
[INFO] Finished at: 2016-09-16T08:56:43+02:00
[INFO] Final Memory: 34M/1189M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal net.wasdev.wlp.maven.plugins:liberty-maven-plugin:1.2.1:start-server (start-server) on project vote-service-wlpcfg: null: MojoExecutionException: NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal net.wasdev.wlp.maven.plugins:liberty-maven-plugin:1.2.1:start-server (start-server) on project vote-service-wlpcfg: null
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:216)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:862)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:286)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:197)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoExecutionException
	at org.codehaus.mojo.pluginsupport.MojoSupport.execute(MojoSupport.java:137)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	... 20 more
Caused by: java.lang.NullPointerException
	at net.wasdev.wlp.maven.plugins.server.StartDebugMojoSupport.writeBootstrapProperties(StartDebugMojoSupport.java:179)
	at net.wasdev.wlp.maven.plugins.server.StartDebugMojoSupport.copyConfigFiles(StartDebugMojoSupport.java:145)
	at net.wasdev.wlp.maven.plugins.server.StartDebugMojoSupport.copyConfigFiles(StartDebugMojoSupport.java:104)
	at net.wasdev.wlp.maven.plugins.server.StartServerMojo.doExecute(StartServerMojo.java:72)
	at org.codehaus.mojo.pluginsupport.MojoSupport.execute(MojoSupport.java:122)
	... 22 more
[ERROR] 

```

--- a/microservice-vote/vote-service-wlpcfg/pom.xml
+++ b/microservice-vote/vote-service-wlpcfg/pom.xml
@@ -13,6 +13,11 @@
     <name>Conference :: Vote :: Config</name>
     <packaging>pom</packaging>
 
+    <properties>
+        <testServerHttpPort>9090</testServerHttpPort>
+        <testServerHttpsPort>9443</testServerHttpsPort>
+    </properties>
+
     <dependencies>
         <dependency>
             <groupId>${project.groupId}</groupId>

